client/src/js/app.js

```diff
@@ -357,6 +352,33 @@ function bhimaConfig($stateProvider, $urlRouterProvider, $urlMatcherFactoryProvi
       data : { label : null }
     })
 
+  
+  /* Fiscal Year */
+  .state('fiscal', {
+    url : '/fiscal/:id',
+    abstract : true,
+    params : {
+      id : { squash : true, value : null }
+    },    
+    controller: 'FiscalController as FiscalCtrl',
+    templateUrl: 'partials/fiscal/fiscal.html'
+  })
+
+    .state('fiscal.list', {
+      url : '',
+      templateUrl : 'partials/fiscal/fiscal.list.html'
+    })
+    .state('fiscal.create', {
+      url : '/create/new',
```

Comment:
@sfount discovered a way to make this URL `/create` instead of `/create/new` today.  Just commenting for future reference.